.circleci/config.yml

```diff
@@ -351,27 +383,30 @@ jobs:
       - run:
           name: embuilder build ALL
           command: |
-            EMCC_CORES=4 python3 ~/emscripten/embuilder.py build ALL
-            python3 ~/emscripten/tests/runner.py test_hello_world
+            EMCC_CORES=4 ~/emscripten/tools/emcoverage.py embuilder build ALL
+            ~/emscripten/tools/emcoverage.py ~/emscripten/tests/runner.py test_hello_world
       - run:
           name: embuilder (LTO)
           command: |
-            EMCC_CORES=4 python3 ~/emscripten/embuilder.py build libcompiler_rt libc libc++abi libc++ libc++-noexcept libal libdlmalloc libpthreads_stub libcompiler_rt_wasm libc_rt_wasm struct_info libc-wasm --lto
-            python3 ~/emscripten/tests/runner.py test_hello_world
+            EMCC_CORES=4 ~/emscripten/tools/emcoverage.py embuilder build libcompiler_rt libc libc++abi libc++ libc++-noexcept libal libdlmalloc libpthreads_stub libcompiler_rt_wasm libc_rt_wasm struct_info libc-wasm --lto
+            ~/emscripten/tools/emcoverage.py ~/emscripten/tests/runner.py test_hello_world
       - run:
           name: embuilder (PIC)
           command: |
-            EMCC_CORES=4 python3 ~/emscripten/embuilder.py build SYSTEM --pic
-            python3 ~/emscripten/tests/runner.py test_hello_world
+            EMCC_CORES=4 ~/emscripten/tools/emcoverage.py embuilder build SYSTEM --pic
+            ~/emscripten/tools/emcoverage.py ~/emscripten/tests/runner.py test_hello_world
       - run:
           name: embuilder (PIC+LTO)
           command: |
-            EMCC_CORES=4 python3 ~/emscripten/embuilder.py build libcompiler_rt libc libc++abi libc++ libc++-noexcept libal libdlmalloc libpthreads_stub libcompiler_rt_wasm libc_rt_wasm struct_info libc-wasm --pic --lto
-            python3 ~/emscripten/tests/runner.py test_hello_world
+            EMCC_CORES=4 ~/emscripten/tools/emcoverage.py embuilder build libcompiler_rt libc libc++abi libc++ libc++-noexcept libal libdlmalloc libpthreads_stub libcompiler_rt_wasm libc_rt_wasm struct_info libc-wasm --pic --lto
+            ~/emscripten/tools/emcoverage.py ~/emscripten/tests/runner.py test_hello_world
       - run:
           name: freeze cache
           command: |
             echo "FROZEN_CACHE=True" >> ~/.emscripten
+      - run:
+          name: upload coverage
+          command: codecov
```

Comment:
Where does this upload to?

Reply:
This uploads the coverage report to codecov so it can be viewed online, in theory. This is mostly just a test. From the look of things, running the tests under coverage is way too slow.